src/Runner.Worker/Handlers/StepHost.cs

```diff
@@ -58,18 +63,26 @@ public Task<string> DetermineNodeRuntimeVersion(IExecutionContext executionConte
             return Task.FromResult<string>(preferredVersion);
         }
 
-        public async Task<int> ExecuteAsync(string workingDirectory,
+        public async Task<int> ExecuteAsync(IExecutionContext context,
+                                            string workingDirectory,
                                             string fileName,
                                             string arguments,
                                             IDictionary<string, string> environment,
                                             bool requireExitCodeZero,
                                             Encoding outputEncoding,
                                             bool killProcessOnCancel,
                                             bool inheritConsoleHandler,
+                                            string standardInInput,
                                             CancellationToken cancellationToken)
         {
             using (var processInvoker = HostContext.CreateService<IProcessInvoker>())
             {
+                Channel<string> redirectStandardIn = null;
+                if (standardInInput != null)
```

Comment:
`!= null`, because `IsNullOrEmpty` would not write `""`, which is _technically_ a valid input (someone could be waiting for it to finish on the other side)

Reply:
I don't understand this comment, could you elaborate more? We should always provide some input right?